website/docs/modeling.mdx

```diff
@@ -0,0 +1,210 @@
+:::
+
+The used type must both be supported by GraphQL Kotlin and Spring Data Neo4j, however, both libaries provide extension mechanisms which allow you to support additional types.
+
+## Functions
+Like properties, functions can be used to expose field in the GraphQL schema.
+
+## Relationships
+
+###  GraphGlue relationships
+
+Declaration of one-sides of relationships
+```kotlin
+@NodeRelationship(label, direction)
+@delegate:Transient
+val propertyName by NodeProperty<NodeType>()
+```
+
+Declaration of many-sides of relationships
+```kotlin
+@NodeRelationship(label, direction)
+@delegate:Transient
+val propertyName by NodeSetProperty<NodeType>()
+```
+
+where `label` is a `String`, and direction is either `Direction.INCOMING` or `Direction.OUTGOING`.
+
+:::caution
+
+Make sure to use `org.springframework.data.annotation.Transient`.
+Kotlin `Transient` does not work and will result in strange errors, as Spring Data Neo4j tries to persist the backing field of the delegated property.
+
+:::
+
+On save, all relationships are saved. Save cascades down added entities, but removed ones.
+Example: If the one side has initially the value `node1`, which then is replaced with `node2`, when saving, `node2` is saved (as it was "added"), while `node1` is not.
+
+
+### Spring Data Neo4j relationships
+
+Declaration of one-sides of relationships
+```kotlin
+@Relationship(label, direction)
+val propertyName: NodeType
+```
+
+Declaration of many-sides of relationships
+```kotlin
+@Relationship(label, direction)
+val propertyName: List<NodeType>
+```
+
+For more information, see [Connecting nodes: @Relationship](https://docs.spring.io/spring-data/neo4j/docs/current/reference/html/#mapping.annotations.relationship)
+
+### Comparison
+
+|| GraphGlue | Spring Data Neo4j |
+|--|--|--|
+|GraphQL representation|One sides are represented by their appropriate type. Many sides are represented by connection types, supporting pagination, filtering and ordering|One sides are represented by their appropriate type. Many sides are represented as GraphQL list, without pagination, filtering or ordering support|
+|Lazy loading|Only lazy loading is supported. Relations are automatically loaded when accessed. Note: when fetching data for GraphQL, the whole subtree is loaded at once using one Cypher query, preventing the n+1 problem.|No lazy loading is supported, all relationships are eagerly loaded, which can result in large subgraphs being loaded. To prevent this, you may use [Projections](https://docs.spring.io/spring-data/neo4j/docs/current/reference/html/#projections)|
+|Matching of opposite sides|Opposite sides are matched if the `label` is the same, but the `direction` the opposite.|Opposite sides are matched if the `label` is the same, but the `direction` the opposite.|
```

Comment:
not a difference?

Reply:
yes, as it is done the same
shall I try to replace it with a single cell with colspan 2?

Reply:
yes? or just make it clear so nobody searches eternities for the differences

Reply:
changed with 06855686233e90623c3d315f6084c5db9ea35252